tools/tertiary-analysis/pyscenic/macros.xml

```diff
@@ -0,0 +1,14 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">
+            aertslab/pyscenic:0.12.1
```

Comment:
the version number should be its own macro and reused in the tools version. https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

This way our bot could bump tool versions ...